src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -149,25 +99,20 @@
                      (nil? (schema/check JRubyInstanceState @%)))
                'JRubyInstanceState))
 
-;; A record representing an individual entry in the JRubyPuppet pool.
+;; TODO: rename, get rid of references to puppet
 (schema/defrecord JRubyPuppetInstance
-                  [pool :- pool-queue-type
-                   id :- schema/Int
-                   max-requests :- schema/Int
-                   flush-instance-fn :- IFn
-                   state :- JRubyInstanceStateContainer
-                   jruby-puppet :- JRubyPuppet
-                   scripting-container :- ScriptingContainer
-                   environment-registry :- (schema/both
```

Comment:
Makes sense to remove this since it's Puppet-specific.  Seems like maybe we'll need a way - as part of the callback initialization hooks we'll do later - to stuff arbitrary custom state onto a JRubyInstance as needed?